staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -100,11 +123,22 @@ func AddFlags(fs *flag.FlagSet) {
 // PrintAndExitIfRequested will check if the -version flag was passed
 // and, if so, print the version and exit.
 func PrintAndExitIfRequested() {
-	if *versionFlag == VersionRaw {
-		fmt.Printf("%#v\n", version.Get())
+	ver := version.Get()
+
+	switch *versionFlag {
+	case VersionTrue:
+		fmt.Printf("Kubernetes %s\n", ver)
+		os.Exit(0)
+	case VersionRaw:
+		fmt.Printf("%#v\n", ver)
+		os.Exit(0)
+	case VersionJson:
+		bytes, _ := json.MarshalIndent(&ver, "", "  ")
```

Comment:
Why not log these errors to stderr?

Reply:
Again, I didn't even think about this. Just copied what was previously done. A separate PR, perhaps(?)

Reply:
Sorry, forget my previous comment, I read your comment in a haste. Yeah, we could do that

Reply:
Fixed in the new version of the PR I just pushed